README.md

```diff
@@ -44,39 +44,20 @@ and for Ivy:
 <dependency org="com.netflix.photon" name="Photon" rev="0.1.1" />
 ```
 
-If you need to download the jars instead of using a build system, create a Maven pom file like the following with the desired version:
-
-```xml
-<?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
-  <modelVersion>4.0.0</modelVersion>
-  <groupId>com.netflix.photon.download</groupId>
-  <artifactId>photon-download</artifactId>
-  <version>1.0-snapshot</version>
-  <name>Simple POM to download Photon and dependencies</name>
-  <url>https://github.com/Netflix/photon</url>
-  <dependencies>
-    <dependency>
-      <groupId>com.netflix.photon</groupId>
-      <artifactId>Photon</artifactId>
-      <version>0.1.1</version>
-      <scope>runtime</scope>
-    </dependency>
-  </dependencies>
-</project>
-```
-
-Then execute:
+If you need to download all dependencies, you just have to run:
 
 ```
-mvn -f photon-download.pom.xml dependency:copy-dependencies
+gradle getDependencies
```

Comment:
MarcAntoine-Arnaud, would it make sense to replace this line with "./gradlew getDependencies"